alacritty/src/event.rs

```diff
@@ -1344,6 +1344,13 @@ impl Processor {
                 GlutinEvent::UserEvent(Event {
                     payload: EventType::CreateWindow(options), ..
                 }) => {
+                    // XXX Ensure that no context is current when creating a new window, otherwise
+                    // it may lock the current context when asking e.g. EGL on Wayland to create
+                    // new context.
```

Comment:
```suggestion
                    // XXX Ensure that no context is current when creating a new window, otherwise
                    // it may lock the current context when asking e.g. EGL on Wayland to create a
                    // new context.
```